src/Sarif/Readers/NotificationLevelConverter.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.CodeAnalysis.Sarif.Readers
+{
+    public class NotificationLevelConverter : JsonConverter
```

Comment:
Done except for the AlgorithmKind (punt)

---
In reply to: [61638443](https://github.com/Microsoft/sarif-sdk/pull/229#discussion_r61638443) [](ancestors = 61638443)